policyengine_us/parameters/gov/states/ma/tax/income/rates/additional.yaml

```diff
@@ -9,13 +9,16 @@ metadata:
       href: https://malegislature.gov/Bills/191/H86
     - title: Form 1 2023 Massachusetts Resident Income Tax (4% Surtax on Taxable Income Over $1 Million)
       href: https://www.mass.gov/doc/2023-form-1-instructions/download#page=3
+    - title: Form 1 2024 Massachusetts Resident Income Tax - 4% Surtax on Taxable Income Over $1 Million
+      href: https://www.mass.gov/doc/2024-form-1-instructions/download#page=3
```

Comment:
add statutory reference

if it states the threshold is inflation-indexed, add uprating